msteams-platform/messaging-extensions/build-bot-based-plugin.md

```diff
@@ -0,0 +1,242 @@
+---
+title: Create bot based Message extension
+author: v-ypalikila
+description: Learn how to build a bot message extension using Teams Developer Portal and Teams Toolkit.
+ms.localizationpriority: medium
+ms.topic: overview
+ms.author: anclear
+ms.date: 09/07/2023
+---
+
+# Extend bot-based message extension as plugin for Copilot
+
+> [!NOTE]
+> Only bot-based search message extension can be extended as a plugin in Copilot.
+
+Microsoft 365 Copilot, powered by an advanced processing and orchestration engine, integrates Microsoft 365 apps, Microsoft Graph, and large language models (LLMs) to transform your words into a potent productivity tool. Although Copilot can utilize apps and data within the Microsoft 365 ecosystem, many users rely on various external tools and services for work management and collaboration. By extending your message extension as a plugin in Copilot, you can enable users to interact with third-party tools and services, thereby unlocking Microsoft 365 Copilot's full potential. You can achieve this extension by developing a plugin or connecting to an external data source.
```

Comment:
Instead of "thereby unlocking Microsoft 365 Copilot's full potential" how about "empowering them to achieve more with Copilot". (Emphasis on the user, not the product)

Reply:
Updated.